decentraland/unity-explorer
# fix: Localization issue makes loading screen stuck

## What does this PR change?
Fix #2050 

When the user had the Windows language in Spanish he was being stuck at loading screen when tried to connect to Decentraland.

### Applied changes
- Grouped all localization assets into a single addressable group.
- Removed the unused localization tables for ES and IT languages (they were not being used at the moment).
- Re-set the default Locale Id (I suspect there was some conflict there) as English in the Localization settings in the Project Settings.

![image](https://github.com/user-attachments/assets/0828d7b9-ad36-4da6-8ad5-9cc9b6a37a80)
![image](https://github.com/user-attachments/assets/88bfc320-5932-47bf-8764-02644285fdaa)

## How to test the changes?
1. Set your OS language to Spanish.
2. Launch explorer alpha build.
3. Check that you can log in and enter the world without problems.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
